### ISSUE TITLE
Typescript now requires argument names in d.ts typedefs

### ISSUE THREAD

**Issue by eaceaser** [fauna/faunadb-js#137]
I'm getting compilation errors on: https://github.com/fauna/faunadb-js/blob/deb752d2ab85617688947b2a0fcc4f79b4d0f409/src/types/PageHelper.d.ts#L11

Looks like we need to specify an argument name for each `Object` typed arg now, like:

```
 each(lambda: (Object) => void): Promise<void>; 
```

I think `Object` may be deprecated too but I'm not 100% sure. Been out of the typescript game for a minute.

**Comment by eaceaser** [comment]
Here's the actual error message:
```
.../node_modules/faunadb/src/types/PageHelper.d.ts
(11,17): error TS7006: Parameter 'Object' implicitly has an 'any' type.
```

**Comment by sprsquish** [comment]
this has been fixed: 7f8b9cc40c632378075db07d2eb5f7a44e828b2a
